managed_vms/cloudsql/src/main/java/com/example/managedvms/cloudsql/CloudSqlServlet.java

```diff
@@ -40,27 +42,33 @@
   @Override
   public void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException,
       ServletException {
+    // encrypt user ip using PBEWithMD5AndDES
+    BasicTextEncryptor encryptor = new BasicTextEncryptor();
+    encryptor.setPassword(Double.toString(10000*Math.random()));
```

Comment:
We should use SecureRandom or at least add a "don't do this" disclaimer. http://stackoverflow.com/a/11052736/101923

Also, 10000 possible passwords isn't all that many. :-(